### PR TITLE
Client-server: Fix upload keys example request

### PR DESCRIPTION
**Pull request by a-andreyev**
Contributes to #1733

### CODE REVIEW

**Comment by turt2live** [comment]
Thanks for finding and fixing this @a-andreyev! We do need signoff and a changelog before it can be merged though, as per our [contributing guidelines](https://github.com/matrix-org/matrix-doc/blob/master/CONTRIBUTING.rst). When you get a chance, please sign off on the changes and include a `.clarification` changelog entry.

**Comment by a-andreyev** [comment]
Got it! Updated the PR. :slightly_smiling_face:

**Comment by uhoreg** [comment]
Thanks for the fix.  Since you're fixing this spot, could you also fix the same issue in `api/client-server/keys.yaml` and `api/server-server/user_keys.yaml` (which will also require a `server_server` changelog file).

**Comment by a-andreyev** [comment]
Thank you! Missed that previously, fixed.

File: changelogs/client_server/newsfragments/2157.clarification [hunk 1]
```diff
@@ -0,0 +1 @@
+Clarify the encryption algorithms supported by the device of the device keys example. Add `v1` and `sha2` to the name strings.
```
Comment (turt2live) [inline changelogs/client_server/newsfragments/2157.clarification]: Probably don't need the second sentence in both changelogs - the first conveys the meaning enough imo.

Comment (a-andreyev) [reply changelogs/client_server/newsfragments/2157.clarification]: Agree, removed the 2nd sentence

**Comment by uhoreg** [comment]
Thank you very much!

**Pull request merged**